config/routes.rb

```diff
@@ -1,7 +1,16 @@
 Rails.application.routes.draw do
+  post '/login', to: 'auth#login', default: {:format => :json} 
   # Defines the root path route ("/")
   root to: 'users#index'
 
+  scope 'api' do
+    get '/posts/:id/comments' => 'api/comments#index', as: :api_post_comments
+    post '/posts/:id/comments' => 'api/comments#create', as: :api_post_comments_create
```

Comment:
- [ ] Please add an API endpoint that returns all comments for a post: `/users/:user_id/posts/:id/comments`